src/components/dropdown/styles.scss

```diff
@@ -26,11 +29,35 @@
       align-items: center;
       justify-content: center;
       width: 100%;
-      height: 24px;
+      height: 44px;
+      cursor: pointer;
+      background-color: $color-white;
+      border: {
```

Comment:
Check my suggestion below, you can remove the `border` property from here and apply it to the list elements below
```suggestion
```